package.json

```diff
@@ -63,6 +63,7 @@
     "regenerator-runtime": "0.13.7",
     "request": "2.88.2",
     "sha256-file": "1.0.0",
+    "sql.js": "^1.5.0",
```

Comment:
Please note all other packages are exact match (means no `^`) can you remove it and reinstall to update yarn.lock? You can use `-E` flag in the yarn command `yarn add -E sql.js`.